sb_vision/distance_finding.py

```diff
@@ -0,0 +1,139 @@
+"""
```

Comment:
I'm not sure I like the name of this file. Whilst this file does partially handle finding the distance, it also handles _lots_ of other things.

Reply:
That's fair, fixed in d40ccf21b70b466a98e664398d17ed8feb4db40c